examples/src/main/scala/org/apache/spark/examples/streaming/RecoverableNetworkWordCount.scala

```diff
@@ -116,7 +116,7 @@ object RecoverableNetworkWordCount {
     val lines = ssc.socketTextStream(ip, port)
     val words = lines.flatMap(_.split(" "))
     val wordCounts = words.map(x => (x, 1)).reduceByKey(_ + _)
-    wordCounts.foreachRDD((rdd: RDD[(String, Int)], time: Time) => {
+    wordCounts.foreachRDD { (rdd: RDD[(String, Int)], time: Time) =>
```

Comment:
While we're here, can this just be `(rdd, time) =>`? and likewise in the following file, maybe others